ingress/ingress-https-setup.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# path to dashboard repo
+export REPO_TOP_LEVEL=$(git rev-parse --show-toplevel)
+# path to cert & keys
+export CERTIFICATE_DIR="/etc/ssl/certs/tekton-dashboard"
```

Comment:
I guess that this directory can be accessible by the admin user.  Should this be executed by this admin user?
The certificate can be generated in any directory.

Reply:
What do you mean by "executed by this admin user"?

Reply:
I should have put "root" user.  I'm not sure if the regular user can create a directory in the /etc directory.